module/data/spell/__core.json

```diff
@@ -232,6 +232,53 @@
 				}
 			]
 		},
+		{
+			"name": "Chill Touch",
+			"source": "PHB",
+			"data": {
+				"duration.units": null,
+				"duration.value": null
```

Comment:
What is the purpose of this? Surely it would be nice to see in the character sheet that it says "Duration: 1 round"?

Reply:
The AEs will inherit the spell duration, making them both temporary and duplicating the status icon. Seeing two chill touch icons appear on a mob will likely confuse casual players (it definitely confuses mine).

They're special durations, one applies for 1 round + 1 turn, the other applies for 1 round, but actually neither of those are true because the spell isn't sensitive to changes in the number of combatants or turn order. If 80 mooks enter the combat before the the source actor, the spell lasts 80 turns longer.

Reply:
Honestly that sounds like a Midi bug then; if the AE is _defined_ with a different duration (especially a times-up special duration), it really shouldn't be overwritten with inherited data.

I should probably test how exactly it currently works.

Reply:
This is fixed, needed an empty duration object in the effects

Reply:
Imo messing with durations is acceptable; it's information hidden by default in the default sheet. Seems like something we're going to need to do pretty often, too